js/PageLevelProgressNavigationView.js

```diff
@@ -77,11 +77,15 @@ export default class PageLevelProgressNavigationView extends Backbone.View {
     this.updateProgressBar();
   }
 
+  get globalsConfig() {
+    return Adapt.course.get('_globals')._extensions._pageLevelProgress;
```

Comment:
```suggestion
    return Adapt.course.get('_globals')?._extensions?._pageLevelProgress;
```
Otherwise the `this.globalsConfig?.` will have already thrown.

Reply:
I didn't do this previously as these objects should always be added via properties.schema when the `schema-defaults` grunt task is run. Even if the grunt tasks didn't add the schema defaults, it would only error if the `_globals` or `_extensions` objects were missing completely.

There are other references in the plugin which would cause an error first and weren't picked up previously, so I assumed we just relied on the grunt task to negate these:
https://github.com/adaptlearning/adapt-contrib-pageLevelProgress/blob/master/templates/pageLevelProgress.jsx#L13
https://github.com/adaptlearning/adapt-contrib-pageLevelProgress/blob/master/js/adapt-contrib-pageLevelProgress.js#L116

Can amend if needed, but was only looking to amend files directly related to this feature request and not fix other issues.

Reply:
Cool, ta.